apps/src/templates/RegionalPartnerSearch.jsx

```diff
@@ -342,17 +343,38 @@ class RegionalPartnerSearch extends Component {
                           }}
                         >
                           <h4>{collection.heading}</h4>
-                          {collection.workshops.map((workshop, index) => (
-                            <div key={index} style={styles.workshop}>
-                              <div>{workshop.workshop_date_range_string}</div>
-                              <div>{workshop.location_name}</div>
-                              <div>{workshop.location_address}</div>
-                            </div>
-                          ))}
+                          <div>
+                            This Regional Partner is not offering{' '}
+                            {collection.name} workshops at this time. Code.org
+                            will review your application and contact you with
+                            options for joining the program hosted by a Regional
+                            Partner from a different region.
+                          </div>
                         </div>
                       );
                     }
-                  })}
+                  } else if (collection.workshops.length > 0) {
+                    // If workshops present for the given course
+                    return (
+                      <div
+                        key={collectionIndex}
+                        style={{
+                          ...styles.workshopCollection,
+                          ...workshopCollectionStyle
+                        }}
+                      >
```

Comment:
Nit: Thoughts about extracting this div since it's used three tiems?

Reply:
Good point! On it